### ISSUE TITLE
Understands only first argument in a callback

### ISSUE THREAD

**Issue by palexvs** [robotdana/leftovers#17]
If a callback has a few methods name Leftovers takes only the first and complains about all others:

Example:
```
class SomeModel < ActiveRecord::Base
  before_save :test1, :test2

  def test1
    1
  end

  def test2
    2
  end
end
```

result:
```
> leftovers 
Not directly called at all:
app/models/some_model.rb:143:7 test2 def test2

how to resolve: https://github.com/robotdana/leftovers/tree/v0.10.0/README.md#how-to-resolve
```

**Comment by robotdana** [comment]
thanks for the report! ill let you know here when this is fixed

**Comment by robotdana** [comment]
oh thanks for the pr :)

**Comment by robotdana** [comment]
it's released as v0.11.0, thanks for your help :)
